mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -397,21 +435,102 @@ object GeneralizedLinearRegression extends DefaultParamsReadable[GeneralizedLine
 
     /** Trim the fitted value so that it will be in valid range. */
     def project(mu: Double): Double = mu
+
   }
 
   private[regression] object Family {
 
     /**
-     * Gets the [[Family]] object from its name.
+     * Gets the [[Family]] object based on family and variancePower.
+     * 1) retrieve object based on family name
+     * 2) if family name is tweedie, retrieve object based on variancePower
      *
-     * @param name family name: "gaussian", "binomial", "poisson" or "gamma".
+     * @param params a GenerealizedLinearRegressionBase object
```

Comment:
typo in GenerealizedLinearRegressionBase; this type of doc doesn't do anything though because the type is already documented. It should say something non-trivial